interfaces/seccomp/template.go

```diff
@@ -763,4 +763,14 @@ fchown32 - -1 g:###GROUP###
 fchownat - - -1 g:###GROUP###
 lchown - -1 g:###GROUP###
 lchown32 - -1 g:###GROUP###
+
+# allow chown to ###GROUP###:###USERNAME###
+# (chown ###GROUP###:###USERNAME###)
```

Comment:
I don't understand the relationship of this and the new test.  Reading the comment here I would have expected (naively? ) this to be about  chown daemon:daemon

Reply:
@pedronis - erf, I somehow pushed an intermediate commit. Since the intended commit invalidates the previous reviews (sorry @zyga and @mvo5), I have rebased and force pushed for clarity of review. I also realized I may as well also add support for 'chown root:snap_daemon'.